src/osm_configurator/model/project/calculation/calculation_manager.py

```diff
@@ -10,39 +28,55 @@ class CalculationManager:
     phases which are also managed by the CalculationController.
     """
 
-    def __init__(self, configuration_manager):
+    def __init__(self, configuration_manager: ConfigurationManager):
         """
         Gets called when we first create an object of this class. It saves all information it needs for
         managing the calculations.
 
         Args:
             configuration_manager (configuration_manager.ConfigurationManager): Saves all information required to configure the calculation.
         """
-        pass
+        self._config_manager = configuration_manager
+        self._phases: List[ICalculationPhase] = [
```

Comment:
this is hardcoded, bad if you want to add a new phase, only do this if there is no other way.
Maybe more as a constant? idk

Reply:
denied change